### PR TITLE
ISPN-11382 Implement the CLI stats command

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/ISPN-11382

### CODE REVIEW

File: cli/cli-client/src/main/resources/help/stats.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+EXAMPLES
+--------
+`stats //containers/default` +
+Displays statistics about the default container.
+
+`stats //containers/default/caches/mycache` +
+Displays statistics about the `mycache` cache.
+
+SEE ALSO
```
Comment (pruivo) [inline cli/cli-client/src/main/resources/help/stats.adoc]: I think it is missing a newline before "SEE ALSO". 
When I run `help stats`, this section is not formatted properly

File: cli/cli-client/src/main/java/org/infinispan/cli/connection/rest/RestConnection.java [hunk 2]
```diff
@@ -511,7 +511,19 @@ public String execute(List<CommandInputLine> commands) throws IOException {
...
+               Resource resource = activeResource;
+               if (command.hasArg(CliCommand.NAME)) {
+                  resource = pathToResource(command.arg(CliCommand.NAME));
+               }
+               if (resource instanceof CacheResource) {
+                  response = client.cache(resource.getName()).stats();
+               } else if (resource instanceof ContainerResource) {
+                  response = client.cacheManager(resource.getName()).stats();
+               } else {
+                  throw MSG.invalidResource(resource.getName());
```
Comment (pruivo) [inline cli/cli-client/src/main/java/org/infinispan/cli/connection/rest/RestConnection.java]: nitpick, when I run `stats` in the root, the error message is `Invalid resource ''`. 
I would suggest replacing the empty string with `/`

Comment (tristantarrant) [reply cli/cli-client/src/main/java/org/infinispan/cli/connection/rest/RestConnection.java]: Done

File: cli/cli-client/src/main/resources/help/stats.adoc (hunk 1 above)
Comment (tristantarrant) [reply cli/cli-client/src/main/resources/help/stats.adoc]: Dibe

**Comment by tristantarrant** [comment]
I've updated the backport too

**Comment by pruivo** [comment]
@tristantarrant looks good to me. 
can I merge it? or have you started the release?

**Comment by pruivo** [comment]
integrated! thanks @tristantarrant !

**Pull request merged**